bundles/org.openhab.binding.opengarage/src/main/resources/OH-INF/i18n/opengarage.properties

```diff
@@ -18,6 +18,16 @@ thing-type.config.opengarage.opengarage.port.label = Port
 thing-type.config.opengarage.opengarage.port.description = Port of the OpenGarage Web API interface.
 thing-type.config.opengarage.opengarage.refresh.label = Refresh Interval
 thing-type.config.opengarage.opengarage.refresh.description = Specifies the refresh interval in seconds.
+thing-type.config.opengarage.opengarage.doorTransitionTimeSeconds.label = Door Transition Time
+thing-type.config.opengarage.opengarage.doorTransitionTimeSeconds.description = Specifies how long it takes the garage door to fully open / close. Include time it takes for OpenHab to emit beeps. Round up.
+thing-type.config.opengarage.opengarage.doorOpeningState.label = Door Opening State
+thing-type.config.opengarage.opengarage.doorOpeningState.description = Text state to report when garage is opening
+thing-type.config.opengarage.opengarage.doorOpenState.label = Door Open State
+thing-type.config.opengarage.opengarage.doorOpenState.description = Text state to report when garage is open (and not transitioning)
+thing-type.config.opengarage.opengarage.doorClosingState.label = Door Closing State
+thing-type.config.opengarage.opengarage.doorClosingState.description = Text state to report when garage is closing
+thing-type.config.opengarage.opengarage.doorClosedState.label = Door Closed State
+thing-type.config.opengarage.opengarage.doorClosedState.description = Text state to report when garage is closed (and not transitioning)
```

Comment:
Please make consistent the descriptions here and in the thing description (and in the documentation). This is not the case.